app/code/community/Pulchritudinous/Queue/etc/worker.xml.sample

```diff
@@ -52,7 +52,23 @@
                 <reschedule>0</reschedule>
                 <!-- Logging destination. The default is pulchqueue.log -->
                 <logfile>unique_worker_name.log</logfile>
+                <!-- Optional! Add this node jobs that should be run based on a defined schedule -->
+                <recurring>
+                    <!--
+                    Add a helper class to evaluate if the recurring job should run or not.
+                    If no class is defied it will be automagically be handled as runnable.
+                    -->
+                    <helper>pulchqueue/my_helper::myMethod</helper>
```

Comment:
If this is only for checking if it is allowed to run or not, it should probably have a more specific name than "helper". Maybe ```<active_check>``` or something along that line?

Reply:
I agree, the name ```helper``` is just a name conversion from the layout system. They have also a helper definition that is also just expecting a single value to be retuned. But sure, this name could be changed to something else. Would just be more convenient if the XML configuration did not have to many unique keys for each element. Like if we would like to implements this "helper" functionality in another node it would be convenient for the developer if the definition is the same in every node so they don't have to check the documentation.